packages/netlify-cms-core/src/components/Collection/Entries/EntriesSearch.js

```diff
@@ -27,10 +27,10 @@ class EntriesSearch extends React.Component {
     searchEntries(searchTerm);
   }
 
-  componentWillReceiveProps(nextProps) {
-    if (this.props.searchTerm === nextProps.searchTerm) return;
-    const { searchEntries } = this.props;
-    searchEntries(nextProps.searchTerm);
+  componentDidUpdate(prevProps) {
+    if (prevProps.searchTerm === this.props.searchTerm) return;
+    const { searchEntries } = prevProps;
+    searchEntries(this.props.searchTerm);
```

Comment:
[Side effects on props change should be in `componentDidUpdate`](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html#side-effects-on-props-change)